src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -55,30 +58,52 @@ public void createFeedbackResponse(FeedbackResponseAttributes fra)
             }
         }
     }
-
+    
     public FeedbackResponseAttributes getFeedbackResponse(
             String feedbackResponseId) {
-        return frDb.getFeedbackResponse(feedbackResponseId);
+        FeedbackResponseAttributes response = frDb.getFeedbackResponse(feedbackResponseId);
+        if (response == null) {
+            return null;
+        }
```

Comment:
This null check is not present in many places. Are those places safe without it?